hack/tools/go.mod

```diff
@@ -1,14 +1,14 @@
 module sigs.k8s.io/cluster-api/hack/tools
 
-go 1.12
+go 1.15
 
 require (
 	github.com/golangci/golangci-lint v1.23.8
 	github.com/onsi/ginkgo v1.12.0
 	github.com/vmware/govmomi v0.23.1
-	k8s.io/code-generator v0.18.0
-	sigs.k8s.io/controller-tools v0.2.9
+	k8s.io/code-generator v0.20.2
+	sigs.k8s.io/controller-tools v0.4.1-0.20201002000720-57250aac17f6
```

Comment:
does the controller-tools need to be the nightly build version as well? I see capi is using v0.5.0

Reply:
Currently, we are pulling in the CAPI sha which was before CAPI moved to go1.16 and k8s1.21. Hence `controller-tools` points to a nightly build. We might have another change to move forward to the latest CAPI version once an early build for version 0.4 gets released.

Reply:
Opening an issue to track this one